### ISSUE TITLE
Brand request: "Deezer" icon update

### ISSUE THREAD

**Issue by gorlanova** [FortAwesome/Font-Awesome#20415]
### Description

Hello !

Deezer had a branding a while ago, and I noticed the icon has not been updated on font-awesome, would be nice to have the new logo in the icon :)

Thanks so much

### Official website

https://www.deezer.com/us/

### Brand guidelines

https://deezerbrand.com/document/37#/-/overview

### Icon request checklist

- [X] The title starts with "Brand request: ", is followed by the requested brand name
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/brands#faqs)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for the heads-up

**Comment by just-maiyak** [comment]
Hello :) Is this one going to be looked into ?

**Comment by sensibleworld** [comment]
Yup! We just released an update with version 7: https://fontawesome.com/icons/deezer?f=brands&s=solid

**Comment by just-maiyak** [comment]
It looks lovely ! Thank you so much 🚀
